quesma/quesma/config/config_v2.go

```diff
@@ -529,6 +582,10 @@ func (c *QuesmaNewConfiguration) TranslateToLegacyConfig() QuesmaConfiguration {
 
 			conf.IndexConfig[indexName] = processedConfig
 		}
+
+		conf.EnableIngest = true
+		conf.IngestStatistics = c.IngestStatistics
```

Comment:
```suggestion
		conf.IngestStatistics = true
```

Reply:
(In my opinion it should be always turned on and not rely on user's configuration (at least yet))